app/controllers/contact_embed_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class ContactEmbedController < ApplicationController
+  rescue_from GdsApi::BaseError do |e|
+    GovukError.notify(e)
+    render "new_api_down", layout: rendering_context
+  end
+
+  def new
```

Comment:
_New_ and _create_ are pretty much synonyms to me

Reply:
They are pretty similar, aren't they! For backend devs, they're a [standard Rails boilerplate](https://edgeguides.rubyonrails.org/getting_started.html#getting-up-and-running), so they do have some specific meaning.